README.md

```diff
@@ -8,4 +8,7 @@ React Native - expo altyapısıyla geliştirilmiş olup iOS ve Andriod uyumludur
 - Run `yarn && expo start`
 
 ## Running tests
-- Run `yarn test`
\ No newline at end of file
+- Run `yarn test`
+
+###
```

Comment:
There is a syntax error here. We need a title like
`### Legal acknowledge`

Reply:
Okey changed